examples/silicon/read_tb.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+# (c) 2015-2018, ETH Zurich, Institut fuer Theoretische Physik
+# Author: Dominik Gresch <greschd@gmx.ch>
+
+import os
+import shutil
+import subprocess
+import numpy as np
+import tbmodels as tb
+import matplotlib.pyplot as plt
+
+if __name__ == "__main__":
+    WANNIER90_COMMAND = os.path.expanduser("~/git/wannier90/wannier90.x")
+    BUILD_DIR = "./build"
```

Comment:
Here, I think we should either actually run the `wannier90` executable, or provide the `tb.dat` file as an example.